<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/src/img/iс.ico</normaloff>:/src/img/iс.ico</iconset>
  </property>
  <widget class="QPushButton" name="btnSaveSettings">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ELEGANCE</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnSaveSettings{
color: #fff;
   border-radius: 15px;
   border: 4px solid rgb(37, 204, 255);
   background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 rgb(138, 42, 255), stop: 1 rgb(214, 90, 255));
}

QPushButton#btnSaveSettings:hover{
background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                    stop: 0 rgb(142, 29, 255), stop: 1 rgb(255, 47, 245));
border: 4px solid rgb(26, 255, 236);
}
</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>256</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rbtnTeleport">
      <property name="font">
       <font>
        <family>ELEGANCE</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(37, 204, 255);
</string>
      </property>
      <property name="text">
       <string>Teleport</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnStandart">
      <property name="font">
       <font>
        <family>ELEGANCE</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(37, 204, 255)</string>
      </property>
      <property name="text">
       <string>Standart movement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtnHV">
      <property name="font">
       <font>
        <family>ELEGANCE</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(37, 204, 255)</string>
      </property>
      <property name="text">
       <string>H. and V. movement</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
